jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/mysql/MySqlMetadataReader.java

```diff
@@ -0,0 +1,40 @@
+package com.exasol.adapter.dialects.mysql;
+
+import java.sql.Connection;
+import java.util.Set;
+
+import com.exasol.adapter.AdapterProperties;
+import com.exasol.adapter.dialects.BaseIdentifierConverter;
+import com.exasol.adapter.dialects.IdentifierConverter;
+import com.exasol.adapter.jdbc.*;
+
+
+public class MySqlMetadataReader extends AbstractRemoteMetadataReader {
```

Comment:
Add javadoc. Use other dialect's comments as templates. 

For example here javadoc should be:
/**
 * Metadata reader that reads MySQL-specific database metadata.
 */

Reply:
Done